### ISSUE TITLE
Fix a11y checks

### ISSUE THREAD

**Issue by nottrobin** [canonical/ubuntu.com#11829]
The a11y checks are always failing (see [the actions tab](https://github.com/canonical-web-and-design/ubuntu.com/actions)). Let's fix them or remove them.

**Comment by anthonydillon** [comment]
I can't see the workflow results for some reason. Do you know why?

**Comment by nottrobin** [comment]
The [workflow page](https://github.com/canonical/ubuntu.com/actions/runs/2918607206) says "No event triggers defined in `on`"- probably a yaml configuration error.
